group/src/main/java/com/android/group/viewmodel/GroupViewModel.java

```diff
@@ -0,0 +1,36 @@
+package com.android.group.viewmodel;
+
+import android.arch.lifecycle.ViewModel;
+import android.util.Log;
+
+import com.android.group.model.Venue;
+import com.android.group.model.firebase.Group;
+
+public class GroupViewModel extends ViewModel {
+
+    private static GroupViewModel singleInstance;
+    private Group currentGroup;
+    private static final String TAG = "GroupViewModel";
+
+    private GroupViewModel(){}
+
+    public static GroupViewModel getSingleInstance() {
```

Comment:
why make this viewmodel a singleton? Won't it be subject to the ViewModel Providers?

Reply:
I think you right on this.